nodejs12.x/cookiecutter-typescript-app-template/{{cookiecutter.project_name}}/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "web_template",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
There isn't a single entry point with this being a serverless app. Should this be left as-is or changed to something else?

Reply:
I think we can leave as it is, or if it is not a required field, we can also remove it.